### DISCUSSION TITLE
Welcome to Knowledge Discussions!

### DISCUSSION THREAD

**Discussion by RobRoyce** [KnowledgeCanvas/knowledge#77] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about why you enjoy using Knowledge. This will help us figure out what features everyone is interested in, and help new users decide if Knowledge is suitable for their needs.

**Comment by karstegg** [comment]
Hi there this is a great project with massive potential.
Are you aware of the Quivr repo, based on semantic search uses langchain, supabase vectorstore and openai embeddings. Could be excellent way of incorporating LLM over KC
